src/controller/auth/signInPage.js

```diff
@@ -0,0 +1,8 @@
+const { join } = require('path');
+
+const getSignInPage = (req, res) => {
+  res.sendFile(
+    join(__dirname, '..', '..', '..', 'public', 'html', 'auth', 'login.html'),
+  );
+};
+module.exports = getSignInPage;
```

Comment:
Separate HTM page controllers into a dedicated 'views' or 'pages' folder to improve code organization and simplify maintenance.